test/e2e/configuration_test.go

```diff
@@ -267,8 +269,9 @@ func TestOperatorProxyConfiguration(t *testing.T) {
 
 	// Wait for the image registry resource to have an updated StorageExists condition
 	// showing that the operator can no longer reach the storage providers api
-	framework.ConditionExistsWithStatusAndReason(te, defaults.StorageExists, operatorapiv1.ConditionUnknown, "Unknown Error Occurred")
+	framework.ConditionExistsWithStatusAndReason(te, defaults.StorageExists, operatorapiv1.ConditionUnknown, "")
```

Comment:
An empty reason?

Reply:
for ConditionExistsWithStatusAndReason it means "I don't care".